resources/schema.gql

```diff
@@ -1163,6 +1164,14 @@ type DatasetData {
 	"""
 	estimatedSize: Int!
 	"""
+	Total number of object links in this dataset
+	"""
+	numObjectLinksTotal: Int!
+	"""
+	An estimated size of object links
+	"""
+	estimatedObjectLinksSize: Int!
```

Comment:
This name is confusing. I'm not intererested in what links weight, I'm interesting in what content weights. The links themselves are already computed